hail/python/hail/docs/_templates/layout.html

```diff
@@ -10,7 +10,7 @@
 {%- set script_files = script_files + ['../../bootstrap.min.js', '_static/toggle.js'] %}
 {%- set css_files = css_files + ['navbar.css', '_static/rtd_modifications.css'] %}
 
-{% if pagename == "annotationdb" %}
+{% if pagename == "annotation_databaseUI" %}
```

Comment:
this filename mixes camelCase and snake_case. In python, we prefer to use snake_case, and this is a doc for the python code, so change this filename to be `annotation_database_ui`.